### ISSUE TITLE
NTR: IL-1 receptor antagonist (IL-1RA) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8431]
New term request IL-1 receptor antagonist (IL-1RA) high in blood (elthat_200518070819): IL1RA; interleukin 1RA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000758
- label: Elevated circulating interleukin 1 receptor antagonist concentration
- parents: [Abnormal immune serum protein physiology (HP:0011111)](https://hpo.jax.org/app/browse/term/HP:0011111)
- pmid: 
- def: The concentration of interleukin 1 receptor antagonist (IL-1RA) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8431
- synonyms:
